### ISSUE TITLE
SRTM data has moved to S3 

### ISSUE THREAD

**Issue by bjones-udtgroup** [meshtastic/meshtastic_linkplanner#18]
I'm trying to set this up locally with an eye towards attempting to add the ability to plot multiple points.
I have some UI experience and access to those with quite a bit.

However, when I try to get the SRTM data the referenced page forwards to a page saying the data is not on s3

That is not an issue but I am not finding any .hgt files there... may be just my ignorance about what I am looking for...

Can you give me a clue?

**Comment by thebentern** [comment]
There is a massive overhaul happening right now. Stay tuned :-)

**Comment by mrpatrick1991** [comment]
Yes - be advised the SRTM dataset in the linked bucket is not going to work currently. A new backend which streams the required tiles from the bucket directly is being developed.

**Comment by mrpatrick1991** [comment]
Added a warning to the docs that my install instructions are incorrect when it comes to the tiles in the mentioned s3 bucket.
